pandas/io/excel/_base.py

```diff
@@ -755,7 +756,8 @@ def _value_with_fmt(self, val):
 
     @classmethod
     def check_extension(cls, ext):
-        """checks that path's extension against the Writer's supported
+        """
+        checks that path's extension against the Writer's supported
         extensions.  If it isn't supported, raises UnsupportedFiletypeError."""
```

Comment:
Since you are changing things here, can you put the ending triple quotes on their own line as well? (other cases below as well)

Reply:
see #31891

Reply:
probably best to merge #31891 first, since less violations there and once merged this diff should look good and any leftover misalignment can be pushed here.

Reply:
done